### PR TITLE
Preparation of F-Droid integration for development version

### PR DESCRIPTION
**Pull request by chrgernoe**
Part one of issue #68. As soon as the PR is approved, I will tag the first development version and open a PR in the F-Droid repository.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -108,3 +128,71 @@ the command:
...
+always be compatible to the previous version, however, if a database
+change is necessary, the user will be informed how he can convert
+e.g. his tour book.
+
+### Dev
+
+The development version contains regular snapshots which may lead to
+unexpected changes without notice. It can be installed alongside the
+official `YacGuide` app and is available at [F-Droid] under the name
+`YacGuide Dev`. It uses a independent internal database, hence e.g.
```
Comment (paetz) [inline CONTRIBUTING.md]: Forgot "n": `... an independent...`

File: drawable_resources/logo_circle_dev.svg [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="1095mm"
+   height="1095mm"
+   viewBox="0 0 1095 1095"
+   version="1.1"
+   id="svg8"
+   inkscape:version="0.92.4 (5da689c313, 2019-01-14)"
+   sodipodi:docname="logo_circle_dev.svg"
+   inkscape:export-filename="/home/chrger/Private/YacGuide/yac_logo_border.png"
```
Comment (paetz) [inline drawable_resources/logo_circle_dev.svg]: Your private user directory does actually not belong to the repo. What's the purpose of this?

Comment (chrgernoe) [reply drawable_resources/logo_circle_dev.svg]: That's just a setting within the file. I try to get rid of this.

File: CONTRIBUTING.md (hunk 1 above)
Comment (chrgernoe) [reply CONTRIBUTING.md]: Will fix this.

**Pull request merged**